dev/unistd.h

```diff
@@ -24,7 +24,10 @@ static inline int clock_gettime(int ignore_variable, struct timespec* tv)
 #define TURN_OFF_FP_FAST __pragma(float_control( precise, on, push )) // Save current setting and turn on /fp:precise
 #define TURN_ON_FP_FAST  __pragma(float_control(pop)) // Restore file's default settings
 
-#include <direct.h> /* for _mkdir and _stat */
+
```

Comment:
any reason these includes are not ad the top?

Reply:
Yes, they can be moved up.

Reply:
Done.